Resources/Storage/XROOTStorage.py

```diff
@@ -1597,3 +1598,32 @@ def getCurrentURL( self, fileName ):
     cwdUrl = result['Value']
     fullUrl = '%s/%s' % ( cwdUrl, fileName )
     return S_OK( fullUrl )
+
+  def constructURLFromLFN( self, lfn, withWSUrl = False ):
```

Comment:
If people run tests, which actually brings up the question of why there is no CI running on the github PRs...